### DISCUSSION TITLE
What does the mrd unit stand for?

### DISCUSSION THREAD

**Discussion by ThomasSlagle** [mtex-toolbox/mtex#998] [Ask Anything] [answered]
Hello,

Referring to plotting the angle distribution for misorientation distribution functions, or mdf, what is the unit that is displayed on the chard, frequency in mrd? I am on familiar with this metric.

From the documentation, there exists this graph below; and I am wondering what the unit is on the y axis. Is it related to percentage, etc?

![image](https://user-images.githubusercontent.com/48171047/116837875-02bcf180-ab81-11eb-96ec-8491b938b06c.png)

**Comment by lfgmorales** [comment, marked as answer]
Hi

MRD here means „multiples of random density“, and described the probability of the misorientantion of two grains to fall on the range between d (delta) g and (delta) d. It is a sort of probability function, but normalized

Cheers

Luiz

Luiz F. G. Morales
ScopeM - ETH Zürich
Sent from iPhone

On 3 May 2021, at 05:00, Thomas_Slagle ***@***.***> wrote:

﻿

Hello,

Referring to plotting the angle distribution for misorientation distribution functions, or mdf, what is the unit that is displayed on the chard, frequency in mrd? I am on familiar with this metric.

From the documentation, there exists this graph below; and I am wondering what the unit is on the y axis. Is it related to percentage, etc?

[image]<https://user-images.githubusercontent.com/48171047/116837875-02bcf180-ab81-11eb-96ec-8491b938b06c.png>

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub<https://github.com/mtex-toolbox/mtex/discussions/998>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/ALE5IN2RPW46G3X6HJK6TRLTLYGVVANCNFSM44ABTDNA>.

[ { ***@***.***": "http://schema.org", ***@***.***": "EmailMessage", "potentialAction": { ***@***.***": "ViewAction", "target": "https://github.com/mtex-toolbox/mtex/discussions/998", "url": "https://github.com/mtex-toolbox/mtex/discussions/998", "name": "View Discussion" }, "description": "View this Discussion on GitHub", "publisher": { ***@***.***": "Organization", "name": "GitHub", "url": "https://github.com" } } ]

**Comment by lfgmorales** [comment]
Hi

MRD here means „multiples of random density“, and described the probability of the misorientantion of two grains to fall on the range between d (delta) g and (delta) d. It is a sort of probability function, but normalized

Cheers

Luiz

Luiz F. G. Morales
ScopeM - ETH Zürich
Sent from iPhone

On 3 May 2021, at 05:00, Thomas_Slagle ***@***.***> wrote:


﻿

Hello,

Referring to plotting the angle distribution for misorientation distribution functions, or mdf, what is the unit that is displayed on the chard, frequency in mrd? I am on familiar with this metric.

From the documentation, there exists this graph below; and I am wondering what the unit is on the y axis. Is it related to percentage, etc?

[image]<https://user-images.githubusercontent.com/48171047/116837875-02bcf180-ab81-11eb-96ec-8491b938b06c.png>

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub<https://github.com/mtex-toolbox/mtex/discussions/998>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/ALE5IN2RPW46G3X6HJK6TRLTLYGVVANCNFSM44ABTDNA>.

[ { ***@***.***": "http://schema.org", ***@***.***": "EmailMessage", "potentialAction": { ***@***.***": "ViewAction", "target": "https://github.com/mtex-toolbox/mtex/discussions/998", "url": "https://github.com/mtex-toolbox/mtex/discussions/998", "name": "View Discussion" }, "description": "View this Discussion on GitHub", "publisher": { ***@***.***": "Organization", "name": "GitHub", "url": "https://github.com" } } ]
